services/director-v2/src/simcore_service_director_v2/modules/db/repositories/clusters.py

```diff
@@ -125,7 +126,7 @@ async def create_cluster(self, user_id, new_cluster: ClusterCreate) -> Cluster:
             new_cluster.owner = user_primary_gid
             new_cluster_id = await conn.scalar(
                 sa.insert(
-                    clusters, values=new_cluster.to_clusters_db(only_update=False)
+                    clusters, values=to_clusters_db(new_cluster, only_update=False)
```

Comment:
i like more this approach of a helper function to convert the model into a db-compatible dict ... what made you change your mind?

Reply:
the fact that the models_library does not know about the settings library.